docs/architecture/adr-009-v6-ics27-msgserver.md

```diff
@@ -0,0 +1,110 @@
+# ADR 009: ICS27 message server addition
+
+## Changelog
+* 2022/09/07: Initial draft
+
+## Status
+
+Accepted, implemented in v6 of ibc-go
+
+## Context
+
+ICS 27 (Interchain Accounts), brought a cross-chain account management protocol built upon IBC. 
+It enabled chains to programatically create accounts on behalf of counterparty chains which may enable a variety of authentication methods for this interchain account.
+The initial release of ICS 27 focused on enabling authentication schemes which may not require signing with a private key, such as via on-chain mechanisms like governance.
+
+Following the initial release ICS 27 it became evident that:
+- a default authentication module would enable more usage of ICS 27
+- generic authentication modules should be capable of authenticating an interchain account registration
+- application logic which wraps ics27 packet sends do not need to be associated with the authentication logic
+
+## Decision
+
+The controller module should be simplified to remove the correlation between the authentication logic for an interchain account and the application logic for an interchain account.
+To minimize disruption to developers working on the original design of the ICS 27 controller module, all changes will be made in a backwards compatible fashion. 
+
+### Msg server
+
+To acheive this, as stated by [@damiannolan](https://github.com/cosmos/ibc-go/issues/2026#issue-1341640594), it was proposed to:
+
+> Add a new `MsgServer` to `27-interchain-accounts` which exposes two distinct rpc endpoints:
+> 
+> - `RegisterAccount`
+> - `SubmitTx`
+
+This will enable any SDK (authentication) module to register interchain accounts and send transactions on their behalf. 
+Examples of existing SDK modules which would benefit from this change include:
+- x/auth
+- x/gov
+- x/group
+
+The existing go functions: `RegisterInterchainAccount()` and `SendTx()` will remain unchanged. 
```

Comment:
Should we possibly change the wording slightly here? I guess its just a bit of a nit, but there has been small changes, just subtle ones that don't have an impact on API consumers.

```suggestion
The existing go functions: `RegisterInterchainAccount()` and `SendTx()` will remain to operate as they did in previous release versions. 
```